sps-team-38/src/main/webapp/index.html

```diff
@@ -2,66 +2,43 @@
 <html>
   <head>
         <meta charset="UTF-8">
-        <title>Volunteer Request Form</title>
+        <title>Home Page</title>
         <link rel="stylesheet" href="style.css">
         <script src="script.js"></script>
         <script src="https://ajax.googleapis.com/ajax/libs/jquery/3.1.0/jquery.min.js"></script>
         <meta name="google-signin-client_id" content="808152198654-dkol2ej7qm1dp21rn41p84sku3ajlpp0.apps.googleusercontent.com">
         <script src="https://apis.google.com/js/platform.js" async defer></script>
   </head>
   <body>
-        <p>
-            <div class="g-signin2" data-onsuccess="onSignIn"></div>
-            <div class="data">
-            <p>Email Address</p>
-            <p id="email" class="alert alert-danger"></p>
+        <div id="loggedOut" style="display: none;">
+            <center><h1>Welcome to <span>"Insert Web App Name Here"</span></h1></center>
```

Comment:
Did some search on <center> tag and it seems it was removed in HTML5, might cause problems in rendering the page on the browser. I think using CSS for this will be best.  Try adding [style = "text-align: center"] in the h1 tag.